src/runtime-rs/crates/runtimes/virt_container/src/container_manager/manager.rs

```diff
@@ -136,16 +136,25 @@ impl ContainerManager for VirtContainerManager {
         let c = containers
             .get(container_id)
             .ok_or_else(|| Error::ContainerNotFound(container_id.clone()))?;
+
+        // skip stopped containers
+        if c.is_stopped().await {
+            return Ok(());
+        }
```

Comment:
Why is this being added ? Is it fixing an actual bug ? Or is it because we expect kill_process() to return an error if the container is stopped ? Something else ? In any case, please add some more details in the changelog or comments.